controller/main.go

```diff
@@ -127,6 +127,7 @@ func main() {
 		disableCertRotation = flag.Bool("disable-cert-rotation", false, "disable automatic generation and rotation of webhook TLS certificates/keys")
 		certDir             = flag.String("cert-dir", "/tmp/k8s-webhook-server/serving-certs", "The directory where certs are stored")
 		certServiceName     = flag.String("cert-service-name", "webhook-service", "The service name used to generate the TLS cert's hostname")
+		loadBalancerClass   = flag.String("lbclass", "", "load balancer class. When enabled, metallb will handle only those services with the given lb class")
```

Comment:
nit: lbclass -> lb-class (seems more consistent with the others)
suggestion for the description: only services whose spec.loadBalancerClass matches the given lb class will be handled. (I think it's worth mentioning that this corresponds to the loadBalancerClass field)